dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.ts

```diff
@@ -0,0 +1,21 @@
+import { TfmFacility } from '@ukef/dtfs2-common';
+import orderBy from 'lodash.orderby';
+import { getCompletedAmendments } from './completed-amendments';
+
+/**
+ * Get the latest completed facility value amendment
+ * @param tfmFacility - The tfm facility
+ * @returns The latest completed facility value amendment
+ */
+export const getLatestCompletedAmendmentToFacilityValue = (tfmFacility: TfmFacility): number | undefined => {
+  const { amendments } = tfmFacility;
+  if (!amendments) {
+    return undefined;
+  }
+
+  const completedAmendments = getCompletedAmendments(amendments);
+
+  const latestAmendmentWithDefinedFacilityValue = orderBy(completedAmendments, ['updatedAt'], ['desc']).find(({ value }) => value);
+
+  return latestAmendmentWithDefinedFacilityValue?.value ?? undefined;
```

Comment:
Maybe could do this instead of explicitly returning undefined?

```ts
if (latestAmendmentWithDefinedFacilityValue) {
  return latestAmendmentWithDefinedFacilityValue.value;
}
```

Reply:
I can't because I get the eslint error 'expected return statement at end of arrow function'

Reply:
What do you think about returning null instead of undefined?